pt_BR.ini

```diff
@@ -597,28 +600,31 @@ written = Escrito em C++ pela velocidade e portabilidade
 [Reporting]
 Bad = Ruim
 FeedbackDelayInfo = Seus dados estão sendo enviados em segundo plano.
-FeedbackDesc = Como está a emulação? Conte para nós e para a comunidade!
+FeedbackDesc = Como está a emulação? Conte para nós!
+FeedbackIncludeScreen = Incluir uma captura de tela
 FeedbackSubmitDone = Seus dados foram enviados.
-FeedbackSubmitFail = Não foi possível enviar dados para o servidor. Tente atualizar o PPSSPP.
+FeedbackSubmitFail = Erro ao enviar dados. Atualize o PPSSPP.
 FeedbackThanks = Obrigado pelo seu feedback!
 Gameplay = Jogabilidade
 Graphics = Gráficos
 Great = Ótimo
-In-game = Jogo inicia apresenta bugs demais pra poder jogar
+In-game = Inicia/Bugs
+In-game Description = Jogo inicia porém é impossível jogar
 Menu/Intro = Menu/Intro
-Menu/Intro Description = "Não pode nem entrar no jogo""
-Nothing = Nada
-Nothing Description = Completamente quebrado
+Menu/Intro Description = Trava no menu/intro
+Nothing = Injogável
+Nothing Description = Impossível jogar
 OK = OK
 Open Browser = Abrir navegador
 Overall = Geral
 Perfect = Perfeita
-Perfect Description = Emulação sem falhas para o jogo inteiro - ótimo!
+Perfect Description = Emulação sem falhas, perfeita!
 Plays = Jogável
-Plays Description = Totalmente jogável, mas pode estar com falhas
-ReportButton = Relatar opinião/sugestão #Relatar o que achou
+Plays Description = Jogável, mas tem falhas
+ReportButton = Relatar opinião/sugestão
```

Comment:
Does this actually fit?  Seems pretty long for a button.

This isn't for suggestions - this allows you to share your compatibility reports with the community.  Once reported, it shows here and compatibility ratings for your settings are available to other users:

http://report.ppsspp.org/game/ULES00541_1.02

PPSSPP developers also sometimes look at this data, but it's really "feedback to the community" not "suggestions".

-[Unknown]